Lib/shutil.py

```diff
@@ -521,7 +549,7 @@ def _rmtree_unsafe(path, onerror):
                     # os.scandir or entry.is_dir above.
                     raise OSError("Cannot call rmtree on a symbolic link")
             except OSError:
-                onerror(os.path.islink, fullname, sys.exc_info())
+                onerror(entry.is_symlink, fullname, sys.exc_info())
```

Comment:
`fullname` is not an argument of `entry.is_symlink`.

Since `entry.is_symlink` is a bound method, the user can't use it for dispatching error handling.
```python
def onerror(func, filename, exc_info):
    if func == os.unlink:
        ...
    elif func == os.rmdir:
        ...
    elif func == ???: # was os.path.islink
        ...
```